index.html

```diff
@@ -55,97 +55,8 @@ <h2 class="secondtitle">I'm a software developer</h2>
-                        <button type="submit" class="button1">See Project</button>
-                    </div>
-                </div>
-            </div>
-            <br>
-        <div class="box cards1">
-                <div class="blank">
-                    <div class="cardscontainers">
-                        <p class="cardtitle">Multi-Post Stories<br>Gain+Glory</p>
-                        <ul class="ul1">
-                            <li class="ruby2">Ruby on rails</li>
-                            <li class="css0">css</li>
-                            <li class="java0">JavScript</li>
-                            <li class="html0">html</li>
-                        </ul>
-                        <button type="submit" class="button1">See Project</button>
-                    </div>
-                </div>
-            </div>
-            <br>
-            <div class="box cards1">
-                <div class="blank">
-                    <div class="cardscontainers">
-                        <p class="cardtitle">Multi-Post Stories<br>Gain+Glory</p>
-                        <ul class="ul1">
-                            <li class="ruby2">Ruby on rails</li>
-                            <li class="css0">css</li>
-                            <li class="java0">JavScript</li>
-                            <li class="html0">html</li>
-                        </ul>
-                        <button type="submit" class="button1">See Project</button>
-                    </div>
-                </div>
-            </div>
-            <br>
-            <div class="box cards1">
-                <div class="blank">
-                    <div class="cardscontainers">
-                        <p class="cardtitle">Multi-Post Stories<br>Gain+Glory</p>
-                        <ul class="ul1">
-                            <li class="ruby2">Ruby on rails</li>
-                            <li class="css0">css</li>
-                            <li class="java0">JavScript</li>
-                            <li class="html0">html</li>
-                        </ul>
-                        <button type="submit" class="button1">See Project</button>
-                    </div>
-                </div>
-            </div>
-            <br>
-            <div class="box cards1">
-                <div class="blank">
-                    <div class="cardscontainers">
-                        <p class="cardtitle">Multi-Post Stories<br>Gain+Glory</p>
-                        <ul class="ul1">
-                            <li class="ruby2">Ruby on rails</li>
-                            <li class="css0">css</li>
-                            <li class="java0">JavScript</li>
-                            <li class="html0">html</li>
-                        </ul>
-                        <button type="submit" class="button1">See Project</button>
-                    </div>
-                </div>
-            </div>
-        </div>
+        <div class="card"></div>
```

Comment:
- [ ]   When I try to load your page on small devices, styling is not being applied to your page. You should fix the issue so that Your popup can also appear on small devices/mobiles similar to the one provided from the Figma design.
 
Figma Design             | Your design
:-------------------------:|:-------------------------:
![image](https://user-images.githubusercontent.com/43877689/175431742-50bc2c99-aaa8-4877-b5cc-f2ff4779843c.png)  |  ![image](https://user-images.githubusercontent.com/43877689/175431437-20bc2e43-2478-4bd4-8858-67334b44267e.png)

Reply:
The layout of the work card is breaking on the smaller screen. Consider fixing it.
![image](https://user-images.githubusercontent.com/33268927/175465007-4733924f-5379-45bf-a431-a19d60edc55d.png)